packages/fela-plugin-bidi/src/index.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+import transformStyle from 'bidi-css-js'
+
+import type DOMRenderer from '../../../flowtypes/DOMRenderer'
+import type NativeRenderer from '../../../flowtypes/NativeRenderer'
+
+type Type = 1 | 2 | 3 | 4 | 5
```

Comment:
I copied this from `fela-plugin-rtl`, but can't say I understand why the type definitions are there

Reply:
Hehe. And I copied that from another plugin that directly uses the renderer. Feels like we can remove those :)